osu.Game/Skinning/SkinManager.cs

```diff
@@ -102,9 +110,15 @@ protected override async Task Populate(SkinInfo model, ArchiveReader archive, Ca
         /// <returns>A <see cref="Skin"/> instance correlating to the provided <see cref="SkinInfo"/>.</returns>
         public Skin GetSkin(SkinInfo skinInfo)
         {
+            if (skinInfo == null)
```

Comment:
is `skinInfo` ever null? even so, there are no null checks in it's usage which would likely result in null reference exception, like here for example: https://github.com/ppy/osu/blob/40c1c6072ed587b0f9f3c8ec6d720117788fd62b/osu.Game/Skinning/SkinManager.cs#L92-L97